web/sites/default/config/core.entity_view_display.group.program.default.yml

```diff
@@ -227,7 +229,6 @@ hidden:
   field_enable_programs_menu_item: true
   field_enable_projects_menu_item: true
   field_enable_public_notices: true
-  field_enable_reports: true
```

Comment:
@jookyg , this is a weird one. For some reason this change unhid the field_enable_reports. We don't want that showing up at the bottom of program pages. Likely just need to move it back to hidden here from /admin/group/types/manage/program/display and then cex -y.